bitmap.js

```diff
@@ -0,0 +1,39 @@
+'use strict'; 
+
+const fileHelper = require('./lib/bitmap-file-helper.js');
+const Bitmap = require('./model/bitmap-constructor.js');
+
+
+let filesToConvert = [`${__dirname}/img/palette-bitmap.bmp`, `${__dirname}/img/optimus-prime.bmp`];
+transformBMP(filesToConvert);
+
+function transformBMP(files) {
+  let colorTransforms = [];
```

Comment:
@brials @remilonwheels @caylazabel good use of an empty array pattern for filling the array in the subsequent `for` loop